documentation/docs/how_tos/12_build_foundry_output.mdx

```diff
@@ -15,8 +15,8 @@ import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 import JavaCode from "../libraries/java/how_to/_12_foundry_output/_code_sample.mdx"
 import JavaOutput from "../libraries/java/how_to/_12_foundry_output/_expected_output.md"
-import NodejsCode from "../_admonitions/_guide_coming_soon.md";
-import NodejsOutput from "../_admonitions/_guide_coming_soon.md";
+import NodejsCode from "../libraries/nodejs/how_to/_12_foundry_output/_code_sample.mdx";
+import NodejsOutput from "../libraries/nodejs/how_to/_12_foundry_output/_expected_output.md";
```

Comment:
Is it not 14 ?

Reply:
Oh, number are not corralled, confusing

Reply:
Yes since we have different examples and different folder structure for the examples they don't match